src/Framework/TestResult.php

```diff
@@ -379,6 +389,8 @@ public function addFailure(Test $test, AssertionFailedError $e, float $time): vo
      */
     public function startTestSuite(TestSuite $suite): void
     {
+        $this->currentTestSuiteIsFailed = false;
```

Comment:
```suggestion
        $this->currentTestSuiteHasFailed = false;
```

Perhaps _has failed_ is better?

Or, for consistency with `lastTestFailed`:

```suggestion
        $this->currentTestSuiteFailed = false;
```

Reply:
Good catch. Either it's "is marked (as) failed" or "has failed". I think just `-Failed' is better as it is shorter and to-the-point